ports/rp2/main.c

```diff
@@ -71,6 +88,25 @@ bi_decl(bi_program_feature_group_with_flags(BINARY_INFO_TAG_MICROPYTHON,
     BINARY_INFO_ID_MP_FROZEN, "frozen modules",
     BI_NAMED_GROUP_SEPARATE_COMMAS | BI_NAMED_GROUP_SORT_ALPHA));
 
+#if MICROPY_HW_USB_VENDOR
+// WebUSB URL Descriptor
+#define URL "creepier-dragonfly-8429.dataplicity.io"
```

Comment:
How does the end-user override this? Is there just one blessed front-end for development via WebUSB?

Reply:
The user cannot override this. It is a temorary URL we're using to test the implementation. It would idealy be replaced with a link to something like a MicroPython page about WebUSB support with a list of supported WebUSB enabled editors.